sources/packages/api/src/database/entities/student-scholastic-standing.model.ts

```diff
@@ -90,4 +96,36 @@ export class StudentScholasticStanding extends RecordDataModel {
     },
   )
   studentAssessment?: StudentAssessment;
+  /**
+   * reference offering id is the offering id of the student application at the
+   * time of scholastic standings submission.
+   */
+  @RelationId(
+    (studentScholasticStanding: StudentScholasticStanding) =>
+      studentScholasticStanding.referenceOffering,
+  )
+  referenceOfferingId?: number;
+  /**
+   * The offering id for the application at the time of scholastic standing submission.
+   * Once the scholastic standing is submitted, if there is a reassessment then the current offering id will be different.
+   */
+  @ManyToOne(() => EducationProgramOffering, {
+    eager: false,
+    cascade: false,
+    nullable: true,
+  })
+  @JoinColumn({
+    name: "reference_offering_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  referenceOffering?: EducationProgramOffering;
+  /**
+   * The number of unsuccessful weeks for a fulltime application that have a scholastic
```

Comment:
Typo here. Please change to "The number of unsuccessful weeks for a fulltime application that have scholastic standings.